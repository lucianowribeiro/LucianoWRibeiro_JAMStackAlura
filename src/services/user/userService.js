import isStagingEnv from '../../infra/env/isStagingEnv';
import HttpClient from '../../infra/http/HttpClient';
import authService from '../auth/authService';

const BASE_URL = isStagingEnv
  ? 'https://instalura-api-git-master-omariosouto.vercel.app'
  : 'https://instalura-api-omariosouto.vercel.app';

const userService = {
  async getProfilePage(context) {
    try {
      const token = await authService(context).getToken();
      console.log(token);
      const response = await HttpClient(`${BASE_URL}/api/users/posts`, {
        headers: {
          Authorization: `Bearer ${token}`,
        },
      });
      return {
        user: {
          likes: '100',
        },
        posts: response.data,
      };
    } catch (err) {
      throw new Error(err);
    }
  },
};
export default userService;
