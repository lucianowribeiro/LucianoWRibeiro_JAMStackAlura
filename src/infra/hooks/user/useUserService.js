import React from 'react';
import userService from '../../services/user/userService';

const useUserService = {
  getProfilePage(password) {
    const [response, setResponse] = React.useState({
      DATA: null,
      LOADING: true,
      ERROR: null,
    });
    React.useEffect(() => {
      userService.getProfilePage(password)
        .then((responseFromServer) => {
          if (responseFromServer) {
            setResponse({
              DATA: responseFromServer,
              LOADING: false,
              ERROR: null,
            });
          } else {
            setResponse({
              DATA: null,
              LOADING: false,
              ERROR: 'Erro ao buscar dados do usuario ,por favor tente novamente',
            });
          }
        });
    }, []);
    return response;
  },
};
export default useUserService;
