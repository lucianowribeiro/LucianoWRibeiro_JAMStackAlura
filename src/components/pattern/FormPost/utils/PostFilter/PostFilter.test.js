import React from 'react';
import user from '@testing-library/user-event';
import { render, screen } from '../../../../../infra/test/testUtils';

import PostFilter from './index';

describe('<PostFilter />', () => {
  describe('renders component', () => {
    test('and select filter', () => {
      const url = 'https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg';
      const setFilter = jest.fn();
      const resetMock = jest.fn();
      const component = render(
        <PostFilter
          url={url}
          setFilter={setFilter}
          filterType="1977"
          reset={resetMock}
        />,
      );
      // select image
      const imageSelected = screen.getByRole('img', {
        name: /filters instalura/i,
      });
      user.click(imageSelected);
      // verify post button
      const postButton = screen.getByRole('button', { name: /postar/i });
      expect(postButton).toBeEnabled();
      //
      expect(resetMock).toHaveBeenCalledTimes(0);
      expect(setFilter).toHaveBeenCalledTimes(1);
      //
      expect(component).toMatchSnapshot();
    });
  });
});
