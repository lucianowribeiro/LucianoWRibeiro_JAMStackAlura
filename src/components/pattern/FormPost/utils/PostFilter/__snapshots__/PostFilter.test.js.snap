// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostFilter /> renders component and select filter 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c1 {
  border: 0;
  cursor: pointer;
  padding: 12px 26px;
  font-weight: bold;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 400;
  line-height: 1;
  -webkit-transition: opacity 200ms ease-in-out;
  transition: opacity 200ms ease-in-out;
  border-radius: 12px;
  width: 100%;
}

.c1:hover,
.c1:focus {
  -webkit-filter: blur(0.9px);
  filter: blur(0.9px);
}

.c1:disabled {
  cursor: not-allowed;
  opacity: 0.5;
}

.c0 {
  border: 0;
  cursor: pointer;
  padding: 12px 26px;
  font-weight: bold;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 400;
  line-height: 1;
  -webkit-transition: opacity 200ms ease-in-out;
  transition: opacity 200ms ease-in-out;
  border-radius: 12px;
}

.c0:hover,
.c0:focus {
  -webkit-filter: blur(0.9px);
  filter: blur(0.9px);
}

.c0:disabled {
  cursor: not-allowed;
  opacity: 0.5;
}

@media only screen and (min-width:0px) {
  .c1 {
    font-size: 12px;
    font-weight: 400;
    line-height: 1;
  }
}

@media only screen and (min-width:768px) {
  .c1 {
    font-size: 16px;
    font-weight: 400;
    line-height: 1.25;
  }
}

@media only screen and (min-width:0px) {
  .c0 {
    font-size: 12px;
    font-weight: 400;
    line-height: 1;
  }
}

@media only screen and (min-width:768px) {
  .c0 {
    font-size: 16px;
    font-weight: 400;
    line-height: 1.25;
  }
}

<body>
    <div>
      <form
        id="filters"
        style="display: flex; flex-direction: column; justify-content: center; align-items: center; height: 40%; width: 100%;"
      >
        <div
          class="flickity-enabled"
          tabindex="0"
        >
          <div
            class="flickity-viewport"
            style="height: 0px;"
          >
            <div
              class="flickity-slider"
              style="left: 0px;"
            >
              <button
                class="c0 is-selected"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-none"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-1977"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-ashby"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-brannan"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-clarendon"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-crema"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-gingham"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-ginza"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-inkwell"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-kelvin"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-lark"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-lo-Fi"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-ludwig"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-maven"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-mayfair"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-moon"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-nashville"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-perpetua"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-poprocket"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-reyes"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-rise"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-sierra"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-skyline"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-slumber"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-toaster"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-valencia"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-vesper"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-walden"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-willow"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
              <button
                aria-hidden="true"
                class="c0"
                mode=""
                style="padding: 0px 3px; position: absolute;"
              >
                <img
                  alt="filters instalura"
                  class="filter-xpro-ii"
                  src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                  style="width: 100px; height: 80px;"
                />
              </button>
            </div>
          </div>
          <button
            aria-label="Previous"
            class="flickity-button flickity-prev-next-button previous"
            disabled=""
            type="button"
          >
            <svg
              class="flickity-button-icon"
              viewBox="0 0 100 100"
            >
              <path
                class="arrow"
                d="M 10,50 L 60,100 L 70,90 L 30,50  L 70,10 L 60,0 Z"
              />
            </svg>
          </button>
          <button
            aria-label="Next"
            class="flickity-button flickity-prev-next-button next"
            disabled=""
            type="button"
          >
            <svg
              class="flickity-button-icon"
              viewBox="0 0 100 100"
            >
              <path
                class="arrow"
                d="M 10,50 L 60,100 L 70,90 L 30,50  L 70,10 L 60,0 Z"
                transform="translate(100, 100) rotate(180) "
              />
            </svg>
          </button>
          <ol
            class="flickity-page-dots"
          />
        </div>
        <button
          class="c1 submitPost"
          form="filters"
          mode=""
          style="width: 80%; margin-top: 2.5rem;"
        >
          Postar
        </button>
      </form>
    </div>
  </body>,
  "container": <div>
    <form
      id="filters"
      style="display: flex; flex-direction: column; justify-content: center; align-items: center; height: 40%; width: 100%;"
    >
      <div
        class="flickity-enabled"
        tabindex="0"
      >
        <div
          class="flickity-viewport"
          style="height: 0px;"
        >
          <div
            class="flickity-slider"
            style="left: 0px;"
          >
            <button
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE is-selected"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-none"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-1977"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-ashby"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-brannan"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-clarendon"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-crema"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-gingham"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-ginza"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-inkwell"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-kelvin"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-lark"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-lo-Fi"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-ludwig"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-maven"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-mayfair"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-moon"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-nashville"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-perpetua"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-poprocket"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-reyes"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-rise"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-sierra"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-skyline"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-slumber"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-toaster"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-valencia"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-vesper"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-walden"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-willow"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
            <button
              aria-hidden="true"
              class="Button__ButtonWrapper-dey3sw-0 hxjHNE"
              mode=""
              style="padding: 0px 3px; position: absolute;"
            >
              <img
                alt="filters instalura"
                class="filter-xpro-ii"
                src="https://sites.google.com/site/minhasemoco/_/rsrc/1342875510991/config/Google%20-%20Google%20Chrome.jpg"
                style="width: 100px; height: 80px;"
              />
            </button>
          </div>
        </div>
        <button
          aria-label="Previous"
          class="flickity-button flickity-prev-next-button previous"
          disabled=""
          type="button"
        >
          <svg
            class="flickity-button-icon"
            viewBox="0 0 100 100"
          >
            <path
              class="arrow"
              d="M 10,50 L 60,100 L 70,90 L 30,50  L 70,10 L 60,0 Z"
            />
          </svg>
        </button>
        <button
          aria-label="Next"
          class="flickity-button flickity-prev-next-button next"
          disabled=""
          type="button"
        >
          <svg
            class="flickity-button-icon"
            viewBox="0 0 100 100"
          >
            <path
              class="arrow"
              d="M 10,50 L 60,100 L 70,90 L 30,50  L 70,10 L 60,0 Z"
              transform="translate(100, 100) rotate(180) "
            />
          </svg>
        </button>
        <ol
          class="flickity-page-dots"
        />
      </div>
      <button
        class="Button__ButtonWrapper-dey3sw-0 jeOWtT submitPost"
        form="filters"
        mode=""
        style="width: 80%; margin-top: 2.5rem;"
      >
        Postar
      </button>
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
